### DISCUSSION TITLE
Wio Terminal with CircuitPython | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#648] [Q&A]
# Wio Terminal with CircuitPython | Seeed Studio Wiki

Wio Terminal with CircuitPython

https://wiki.seeedstudio.com/Software-FreeRTOS/

**Comment by rmrfus** [comment]
From CircuitPython page: "NOTE: The RTL8720DN isn’t supported from CircuitPython. Therefore, Wifi and Bluetooth will not work."

**Reply by lindgrenar** [reply]
Luckily the hardware isn't called something ironic like the "wireless input output terminal", that would sure make this reach new levels of dumb!

/s
